src/services/lista-nominal/ListaNominal.ts

```
@@ -0,0 +1,125 @@
+export const buildUrlWithParams = (
+  baseUrl: string, 
+  params?: {
+    sorting?: SortingItem[],
+    filters?: FilterItem,
+    pagination?: Pagination;
+    listName: string,
+    ine?: string,
+    municipio_id_sus: string,
+    search?: string;
+  }
+): string => {
```

Biome (1.9.4)

[error] Excessive complexity of 16 detected (max: 15).

Please refactor this function to reduce its complexity score from 16 to the max allowed complexity 15.

(lint/complexity/noExcessiveCognitiveComplexity)